shell/plugins/steve/__tests__/getters.spec.js

```diff
@@ -0,0 +1,40 @@
+import _getters from '@shell/plugins/steve/getters';
+
+const { urlFor } = _getters;
+
+describe('dashboard-store: getters', () => {
+  describe('dashboard-store > getters > urlFor', () => {
+    // we're not testing function output based off of state or getter inputs here since they are dependencies
+    const state = { config: { baseUrl: 'protocol' } };
+    const getters = {
```

Comment:
Similar consideration done for the actions are worth also for this case. Assignment done outside of a `beforeEach` lead to issue with mutated objects.

Reply:
https://github.com/jest-community/eslint-plugin-jest/blob/v24.4.0/docs/rules/no-hooks.md